server/events/vcs/gitlab_client.go

```diff
@@ -182,7 +182,33 @@ func (g *GitlabClient) PullIsMergeable(repo models.Repo, pull models.PullRequest
 	if err != nil {
 		return false, err
 	}
-	if mr.MergeStatus == "can_be_merged" && mr.ApprovalsBeforeMerge <= 0 {
+
+	// Get project configuration
+	project, _, err2 := g.Client.Projects.GetProject(mr.ProjectID, nil)
```

Comment:
Why calling it err2?, I think you can leave it as `err` and it will work just fine.